pkgs/development/compilers/rust/default.nix

```diff
@@ -29,7 +29,7 @@ in rec {
       ./patches/disable-test-inherit-env.patch
     ];
 
-    forceBundledLLVM = true;
+    withBundledLLVM = false;
```

Comment:
@LnL7 @dtzWill anyone who does remember why we switched away from this option? I thing this option was true at some point.

Reply:
It broke the tests back then https://github.com/NixOS/nixpkgs/commit/7bd191df6a861e02d5bf2b36a03f6dc70e3e6bfd
It could be still ok though:

- Archlinux does this as well: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/rust 
- Ubuntu uses the bundled one: https://git.launchpad.net/ubuntu/+source/rustc/tree/debian/control?h=ubuntu/cosmic
- Opensuse does: https://build.opensuse.org/package/view_file/devel:languages:rust/rust/rust.spec?expand=1

Since you deal with rust backports sometimes for firefox what would you prefer @andir ?
I guess in the worst case it should be still possible to just re-enable the bundled version in backported version.

Reply:
I really have no strong opinion here. I would certainly like to avoid a situation where we have to port LLVM whenever we have to back port rustc for Thunderbird/Firefox.

How much is the speedup of compilation time? If it is significant I could see this being a good thing in general.

Reply:
Compiling rustc with the bundled llvm takes about 1h05 for llvm and 55 min for rustc. So this halves compilation time.
Note that fedora disables codegen tests when using the bundled llvm.

Reply:
At the moment we do not run tests anyway.

Reply:
Is everything from their llvm fork upstream now? That was the main reason for this IIRC.

Reply:
They only seem to merge pull request from changes that have been before in llvm.

Reply:
Great, I tried to split this out into 2 separate derivations at some point but not rebuilding llvm is even better.